pages/getting-started/understand-odata-in-6-steps.html

```diff
@@ -1355,10 +1355,10 @@ <h4>Contribute to "Understanding OData in 6 steps"</h4>
 <script>
     $(document).ready(function() {
        $('.modal').on('shown.bs.modal', function () {
```

Comment:
Bind to modal dismiss event and remove the trapped focus. 
Also experiment with this pattern an see if it gives a better output

https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html

Reply:
@marabooy  why bind to modal dismiss event? the interest here it to never lose focus on any focusable element within the modal when the modal is open.

Reply:
@marabooy  https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/js/dialog.js this is from the link you sent.